### PR TITLE
Newq programming question1

### PR DESCRIPTION
**Pull request by Anamika2716**
Added one programming question

### CODE REVIEW

File: .idea/InterQues.iml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (arpansac) [inline .idea/InterQues.iml]: Remove this file from the commit

File: .idea/encodings.xml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (arpansac) [inline .idea/encodings.xml]: Remove this file

File: programming-editorials/index.html [hunk 3]
```diff
@@ -9,6 +9,8 @@
         gtag('js', new Date());
         gtag('config', 'UA-157837769-1');
         </script>
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
```
Comment (arpansac) [inline programming-editorials/index.html]: We're not using any css/bootstrap in this project :)

File: programming-editorials/index.html [hunk 4]
```diff
@@ -19,10 +21,36 @@
...
             <img src="../assets/CN_logo.png" height='72px' alt="Coding Ninjas Logo">
+
+
+       <div class="container pt-3">
+
+          <div class="panel panel-default" id="q">
+          <a href="https://www.geeksforgeeks.org/count-the-number-of-contiguous-increasing-and-decreasing-subsequences-in-a-sequence/" >Count the number of contiguous increasing and decreasing subsequences in a sequence</a>
+
+         <div id="description">
+         For a given distinct integer sequence of size N, the task is to count the number of contiguous increasing subsequence and contiguous decreasing subsequence in this sequence.
```
Comment (arpansac) [inline programming-editorials/index.html:35]: Interesting, can you please put the answer in a separate file. Please refer to the question from javascript

**Review by arpansac — CHANGES_REQUESTED** [review]
Please resolve as per the comments

File: .idea/InterQues.iml (hunk 1 above)
Comment (Anamika2716) [reply .idea/InterQues.iml]: Updated code as per comments

File: README.md [hunk 5]
```diff
@@ -56,11 +56,17 @@ For making any changes (from your local system), just following the following st
...
+
 ## Support
 - Send an email directly to arpan(at)codingninjas(dot)in
 
 ## Code of conduct
 
 Respect fellow developers, everyone might be at a different stage of skill levels 
  Encourage fellow developers, pass on an opportunity to them, if they've asked something simple, just point them to the right direction in as polite a manner as possible 
- Thank fellow developers if they help you out in any way 
+ Thank fedllow developers if they help you out in any way
```
Comment (shivamyth) [inline README.md:72]: Change it.

**Review by shivamyth — CHANGES_REQUESTED** [review]
Take a fresh pull and update the PR, there are many issues in it highlighted by @arpansac resolve them also.

**Comment by shivamyth** [comment]
You haven't changed anything in the last 3 days to this PR, so I am closing it

**Pull request closed without merging**